index.js

```diff
@@ -309,7 +313,7 @@ module.exports = createStore =>
 				);
 			}
 
-			if (mounts.has(mountPath)) {
+			if (typeof mounts[mountPath] !== 'undefined') {
```

Comment:
Let's be explicit about assuming integrity of `mounts` and do `mounts.hasOwnProperty(mountPath)`